solutions/usaco-994.mdx

```diff
@@ -2,72 +2,110 @@
 id: usaco-994
 source: USACO Gold 2020 January
 title: Farmer John Solves 3SUM
-author: Qi Wang
+author: Qi Wang, Ryan Chou
 ---
 
 [Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_threesum_gold_jan20.html)
 
+## Explanation
+
+Although the problem may be daunting at first, it helps to break down the problem into separate components.
```

Comment:
wdym separate components

Reply:
help i don't know how to introduce the idea